lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -255,36 +264,56 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
         });
     }
 
-    private updateForm() {
+    private updateForm(): Date {
         const {entry} = this.data.node;
-        let expiryDate = null;
+        this.expiryDate = null;
```

Comment:
Just want to confirm - do you really need to have expiryDate on class level (referencing by this). Is not let enough? I can see you refer to this.expiryDate only in that function